go-controller/pkg/ovn/base_network_controller.go

```diff
@@ -603,3 +618,18 @@ func (bnc *BaseNetworkController) WatchNodes() error {
 	}
 	return err
 }
+
+func (bnc *BaseNetworkController) recordNodeErrorEvent(node *kapi.Node, nodeErr error) {
+	if bnc.IsSecondary() {
+		// TBD, no op for secondary network for now
+		return
+	}
```

Comment:
I think the caller of this will already propagate the error up the stack to be logged, however we need to fix the log as it is an Info instead of Error:

```
func (h *secondaryLayer3NetworkControllerEventHandler) AddResource(obj interface{}, fromRetryLoop bool) error {
	switch h.objType {
	case factory.NodeType:
		node, ok := obj.(*kapi.Node)
		if !ok {
			return fmt.Errorf("could not cast %T object to *kapi.Node", obj)
		}
		var nodeParams *nodeSyncs
		if fromRetryLoop {
			_, nodeSync := h.oc.addNodeFailed.Load(node.Name)
			_, clusterRtrSync := h.oc.nodeClusterRouterPortFailed.Load(node.Name)
			nodeParams = &nodeSyncs{syncNode: nodeSync, syncClusterRouterPort: clusterRtrSync}
		} else {
			nodeParams = &nodeSyncs{syncNode: true, syncClusterRouterPort: true}
		}

		if err := h.oc.addUpdateNodeEvent(node, nodeParams); err != nil {
			klog.Infof("Node add failed for %s, will try again later: %v",
				node.Name, err)
			return err
		}
```